azure-pipelines.yml

```
@@ -78,6 +78,7 @@ jobs:
                   DEALLOCATE db;
                 '@
               displayName: Cleanup databases
+            - powershell: "gci env:"
```

```suggestion
            - powershell: "Get-ChildItem env:"
```
Good practice to avoid aliases in scripts. They're not as stable as the actual cmdlet names.